frontend/src/patientApp/timeOfTest/PatientFormContainer.tsx

```diff
@@ -89,9 +92,11 @@ const PatientFormContainer = () => {
             patient={{
               ...patient,
               facilityId:
-                patient.facility === null ? null : patient.facility?.id,
+                patient?.facility === null
+                  ? null
+                  : (patient?.facility?.id as string),
```

Comment:
I think it's usually safer to cast to string in this way, rather than using `as`

```suggestion
                  : String(patient?.facility?.id),
```

Alternatively, could we fix the type of `patient.facility` to understand that `id` is in fact a string?

Reply:
I hope yes, it's in my list of fixes.